python/lsst/afw/image/image/fitsIoWithOptions.py

```diff
@@ -25,6 +25,7 @@
 import lsst.geom
 from lsst.log import Log
 from lsst.afw.fits import ImageWriteOptions
+from lsst.daf.base import PropertySet
```

Comment:
This line is not needed if you use the dict compatibility interface of PropertySet